powersimdata/scenario/create.py

```diff
@@ -49,8 +57,23 @@ def __init__(self, scenario):
                 ("engine", ""),
             ]
         )
+        self.exported_methods = set(default_exported_methods)
         super().__init__(scenario)
 
+    def __getattr__(self, name):
+        if self.builder is not None and name in self.builder.exported_methods:
+            return getattr(self.builder, name)
+        if self.builder is None and name in _Builder.exported_methods:
+            raise AttributeError(
```

Comment:
I would print: Call `set_grid` first to access {name} method

Reply:
Replacing all of the existing AttributeError test?

Reply:
Not sure I understand. I was just suggesting to replace the error message 
```
f"Create object without a grid set has no attribute {name}. "
 "Did you forget to run set_grid?"
```
with:
```
f"Call set_grid first to access {name} attribute" 
```

Reply:
Done.